pkg/sensors/manager.go

```diff
@@ -47,6 +48,8 @@ func StartSensorManager(bpfDir, mapDir, ciliumDir string) (*Manager, error) {
 				err = handler.addTracingPolicy(op)
 			case *tracingPolicyDel:
 				err = handler.delTracingPolicy(op)
+			case *tracingPolicyList:
+				err = handler.listTracingPolicies(op)
```

Comment:
note `tetra tracingpolicy list` returns me just policies that I added with 'tetra tracingpolicy add ...'
not the one specified with --config-file option. Not sure it's intended

Reply:
Also in this context addTracingPolicy() is incrementing a policy ID but what about --config-file policy ? that should also be accounted?

On another note just to ensure I'm not mixing things up, this is not the ID that should identify a policy in json events right? I remember there was talk about it... that one should be a non recyclable ID or uuid...

Reply:
So the problem here is that we don't use the sensor manager to load the policy provided by the config file. We should fix this, so I made an issue: https://github.com/cilium/tetragon/issues/784.

Note that the same is true for the "base" sensor, but I think that's fine.